perfkitbenchmarker/providers/aws/aws_disk.py

```diff
@@ -76,6 +113,14 @@ def __init__(self, disk_spec, zone):
     self.device_letter = None
     self.attached_vm_id = None
 
+    if self.disk_type != disk.LOCAL:
+      self.metadata = DISK_METADATA[self.disk_type]
+    else:
+      if machine_type[:2].lower() in LOCAL_HDD_PREFIXES:
```

Comment:
please make this if check into a function and define the function right at the place where LOCAL_HDD_PREFIXES are defined.